testing/constraints-3.6.txt

```diff
@@ -0,0 +1,12 @@
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
+# List *all* library dependencies and extras in this file.
+# Pin the version to the lower bound.
+#
+# e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
+# Then this file should have foo==1.14.0
+google-api-core==1.22.2
+libcst==0.2.5
```

Comment:
Please remove `libcst` from here and setup.py, it is not needed for new libraries,.

Reply:
Done.